src/ProcessSlave.php

```diff
@@ -338,7 +338,7 @@ function ($controller) {
                 $httpServer->listen($this->server);
                 if ($this->isLogging()) {
                     $httpServer->on('error', function (\Exception $e) {
-                        error_log(sprintf('Worker error while processing the request. %s: %s', get_class($e), $e->getMessage()));
+                        error_log(\sprintf('Worker error while processing the request. %s: %s', get_class($e), $e->getMessage()));
```

Comment:
error_log could also be prefixed

Reply:
Thanks for pointing that one out, prefixed it.